pyseext/js/PySeExt.GridHelper.js

```diff
@@ -61,52 +61,51 @@ globalThis.PySeExt.GridHelper = {
      * @return {Element} The element for the found row.
      */
     getRow: function(gridSelector, rowData) {
-        var row,
+        var me = this,
             grids = globalThis.Ext.ComponentQuery.query(gridSelector),
             grid,
-            store,
-            foundIndex,
-            prop;
+            store;
 
         if (grids && grids.length) {
             grid = grids[0];
             store = grid.getStore();
 
-            if (typeof(rowData) === 'number') {
-                foundIndex = rowData;
+            row = me.__findRowRecord(rowData, store);
+        }
+        if (row !== undefined && row !== null) {
```

Comment:
Can just use `if (row) {` since both undefined and null are falsey in JavaScript.

Reply:
Updated accordingly